app/components/ChartSettings/index.js

```diff
@@ -21,14 +22,25 @@ class ChartSettings extends Component {
     return -1 !== this.state.modules.indexOf(name);
   }
 
+  _toggleModuleCallback(expanded, name) {
+    if (expanded) {
+      this.setState({
+        activeSettings: name,
+      });
+    }
+  }
+
   _renderModule(name) {
     if (!this._hasModule(name)) {
       return null;
     }
 
     // Setup props, handling special cases for Metadata and ColorPalette
     const moduleProps = update({}, {
-      defaultExpand: { $set: this._shouldDefaultExpand() },
+      defaultExpand: { $set: this._shouldDefaultExpand(name) },
```

Comment:
I think it makes sense to use this prop for both open by default and open because toggled, like you set up, but "default" doesn't really make sense anymore. Maybe just `expanded`?